Slim/Formats/Playlists/CUE.pm

```diff
@@ -337,9 +772,117 @@ sub parse {
 		return;
 	}
 
+        #dump $tracks;
 	return $tracks;
 }
+sub _addCommand{
+        my $cuesheet        = shift;
+	my $tracks          = shift;
+        my $inAlbum         = shift;
+        my $currtrack       = shift;
+        my $command         = shift;
+        my $value           = shift;
+        
+        if ($inAlbum and !defined $cuesheet->{$command}){
+            $cuesheet->{$command} = $value;
+        } elsif (defined $currtrack and !defined $tracks->{$currtrack}->{$command}){
+            $tracks->{$currtrack}->{$command} = $value;
+        }
+        return ($cuesheet,$tracks);
+}
+sub _isRemCommandAccepted {
+        
+        my $remCommand  = shift;
+
+        for my $attribute (@refusedRemCommands){
+
+            if ($remCommand eq $attribute){
+              
+                return 0;
+            }
+        }
+        # we don't want to accept from a REM a command to be ignored.
+        if (_isCommandToIgnore($remCommand)){
+        
+            return 0;
+        }
+        return 1;
+}
+sub _isCommandAccepted{
+        my $command  = shift;
+    
+        if (!_isStandardCommand($command)){
+        
+            return 0;
+        }
+        if (_isCommandToIgnore($command)){
+        
+            return 0;
+        }
+        return 1;
+}
+sub _isStandardCommand{
+        my $command  = shift;
+
+        for my $attribute (@standardCueCommands){
+
+            if ($command eq $attribute){
+                return 1;
+            }
+        }
+        return 0;
+}
+sub _isCommandToIgnore{
+        my $command  = shift;
+
+        for my $attribute (@refusedCueCommands){
 
+            if ($command eq $attribute){
+            
+                return 1;
+            }
+        }
+        return 0;
+}
+sub _getCommandFromLine{
+        my $line  = shift;
+        
+        if ($line =~ /^\s*(\S+)\s+(.*)/i){
+
+            return ($1,$2);
+        }
+        return (undef,undef);
+
+}
+sub _getRemCommandFromLine{
+        my $line  = shift;
+        if ($line =~ /^\"(.*)\"/i){ #"
+
+            return (undef,$1);
+
+        }elsif ($line =~ /^\s*(\S+)\s+(.*)/i){
+
+            return ($1,$2);
+        }
+        return (undef,undef);
+
+}
+sub _removeQuotes{
+        my $line  = shift;
+
+        if ($line =~ /^\"(.*)\"/i){ #"
+            
+            return ($1);
+        }
+        return $line;
+}
```

Comment:
Could you replace this with 

```perl
$line =~ s/^\"(.*?)\".*/$1/i;
return $line;
```

The condition is not required as in case of a mismatch the $line would not be altered. I added the question mark to make this search non-greedy: the match would stop at the first quote.